Ipanema mixed mode

hellt opened the pull request:
This lab demonstrates a joint integration between Nuage Networks VNS and Infovista Ipanema solutions. The difference this lab has from the `0800-VNS-IPANEMA_WAN_OPT` lab, is that it demonstrates how the Nuage Networks VNS can integrate with Infovista Ipanema, where existing sites are already using Ipanema Physical Appliances, hence the mixed deployment model name with virtual and physical Ipanema appliances.

hellt committed:
added setup script

hellt committed:
added gitignore

hellt committed:
added lab files for ipanema mixed mode use case

hellt committed:
switch to lucidchart image links

hellt committed:
added ipanema mixed mode to the labs catalog

hkhazaal committed:
Updated README.md

Minor text edits

hellt committed:
Merge pull request #6 from hkhazaal/patch-1

Updated README.md

hellt committed:
fix url address

hellt committed:
resolve merge conflict

hellt committed:
fix typo

hellt committed:
a nicer wording for main readme file

hellt commented:
do not merge yet, I am considering adding some more installation instructions to add

apooniajjn commented on 0801-VNS-IPANEMA_WAN_OPT_MIXED_MODE/README.md:
```diff
@@ -0,0 +1,91 @@
+
+# Deployment
+The lab deployment process is powered by the [nuxctl](https://nuxctl.nuagex.io) CLI tool. All of the infrastructure deployment will be completed after a user runs the the tool against the [lab template](nuxctl_0801-vns-ipanema_wan_opt_mixed_mode.yml) supplied within this repo.
+
```
Can you tell end user to install nuxctl first before proceeding to lab deployment steps

hellt replied:
yes, absolutely valid. I have it added for viavi lab and I plan to add it to this PR as well

hellt committed:
remove unnecessary variable

hellt committed:
change vsc certs generation logic

support for subsequent runs without failing with an error that a user has been added to the DB

hellt committed:
remove jumpbox address variable

hellt committed:
renamed lab name

hellt committed:
enhanced readme

hellt commented:
@hkhazaal @apooniajjn 
I have removed WIP status and augmented the README file with more details on the prerequisites. 

You can check the changes I made from the last commit https://github.com/nuagex/nuagex-labs/pull/7/commits/5353c6e4b13dab6114363c58887dc0fd1b232520

I also tested the lab deployment using the `CATS in a container` approach and validated it.

My next plan is to change the `ipanema hosted vnf lab` to containerized CATS and then the Ipanema automation artefacts could be considered complete

hellt committed:
fix numbering

hellt committed:
update login to vsd keyword to use keys
